backend/controller/sql/conn.go

```diff
@@ -5,11 +5,19 @@ import (
 	"errors"
 	"fmt"
 
+	cronjobssql "github.com/TBD54566975/ftl/backend/controller/cronjobs/sql"
+	leasessql "github.com/TBD54566975/ftl/backend/controller/leases/sql"
+
 	"github.com/jackc/pgx/v5"
 )
 
+type cronjobsQuerier = cronjobssql.Querier
```

Comment:
Can we remove these? It's only 4 characters shorter but creates unnecessary indirection.

Reply:
This was necessary to work around the duplicate name error: https://stackoverflow.com/questions/41951172/embed-two-structs-with-the-same-name-in-a-struct

Reply:
Ah makes sense. Let's add a comment noting that.

Reply:
Ahh good call, done!